src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
There is an empty line at the beginning of the class definition which should be removed according to the checklist.